front/src/common/rollingStockContext.tsx

```diff
@@ -0,0 +1,47 @@
+import { createContext, useContext, useEffect, useMemo, useState } from 'react';
+
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import type { LightRollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+
+type RollingStockContextValue = {
+  rollingStockMap: Map<string, LightRollingStockWithLiveries>;
+} | null;
```

Comment:
Can we use `Map<string, LightRollingStockWithLiveries> | null` directly instead of an object with a single field?